default_config.toml

```diff
@@ -11,7 +11,7 @@
 ## set the default number of shots/runs used
 ## to determine expectation value statistics
 [main]
-shots = 0
+shots = 1
```

Comment:
Should this be 1000 or 1024? Unsure of whether to use a power of 2 or not :laughing: